iywstbdywdiy/sheeper.py

```diff
@@ -8,11 +8,12 @@
 
 
 class Generator:
-    def __init__(self, output, resize_max=500, search_path=None):
+    def __init__(self, output, resize_max=500, search_path=None, json_path=None):
         self.resize_max = resize_max
         self.test_stamps = []
         self.train_stamps = []
         self.output = output
+        self.json_output = output if json_path is None else json_path
```

Comment:
we could rename `output` and `json_output` to `output_path` and `json_output_path`, couldn't we?

Reply:
This makes the command line arguments quite long, but might still be good.